curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-introductory-javascript-by-building-a-pyramid-generator/660f280dda5040f707c76b4a.md

```diff
@@ -38,6 +38,12 @@ You should pass `i` to your `.repeat()` method.
 assert.match(__helpers.removeJSComments(code), /character\.repeat\(\s*i\s*\)/)
 ```
 
+You should use the `.repeat()` method in the `.push()` method
+
+```js
+assert.match(__helpers.removeJSComments(code), /rows\.push\(\s*character\.repeat\(\s*i\s*\)\)/\s*);
```

Comment:
```suggestion
assert.match(__helpers.removeJSComments(code), /rows\.push\(\s*character\.repeat\(\s*i\s*\)\s*\)/);
```

Here's what I think Naomi meant to type. The regex character needs to be inside of the expression.

Reply:
Ah! I see